go-controller/pkg/util/ovs.go

```diff
@@ -523,7 +523,7 @@ func RunOVNSBAppCtl(args ...string) (string, string, error) {
 	var cmdArgs []string
 	cmdArgs = []string{
 		"-t",
```

Comment:
could you fix the:
`// RunOVNSBAppCtl runs an 'ovn-appctl -t sbdbCtlSockPath command'.`
doc string here?